src/handlers/localstorage/ens.ts

```diff
@@ -0,0 +1,20 @@
+import { getGlobal, saveGlobal } from './common';
```

Comment:
wondering about the need of saving this info, if this is it to make it faster makes sense but I'm a bit worried that we could be showing information that's not actually true if some of these values change on chain.

went to the react query docs to see what's `setQueryData` that I see it's being used with this below, seems like is cached response to the requests with a cache time of 5 min, but the data we're setting is coming from localstorage so the cache time wouldn't work here?

i'm including @jinchung into this comment since I'm not really sure which info we usually store, seems like users searching for random profiles will save info that they won't see ever again but we will be persisting

Reply:
yeah, the cache time wouldn't work here as this is a workaround since we don't have the RQ async storage persist plugin set up, however, I think caching this data is okay since it will always revalidate in the background (which is pretty fast anyway) and I don't think the ens profile data (i.e. records) will change constantly anyway?